### ISSUE TITLE
Incorrect Flagging of Fallthrough

### ISSUE THREAD

**Issue by lawrence-dol** [denoland/deno_lint#746]
When braces are used around a switch's case block, and the block is empty, dlint incorrectly flags the case as a fallthrough. Also, note that the same error is reported twice, which is a separate bug.

### Lint Name

no-fallthrough

### Code Snippet

```js
switch(somevalue) {
    case 0: {
        // nothing to do for this value
        } break;
    case 1: {
        // nothing to do for this value
        } break;
    }
```

### Expected Result

No errors.

### Actual Result

```
error[no-fallthrough]: Fallthrough is not allowed
 --> test.js:2:5
38;5;12m  |
2 |       case 0: {
  |  _____^
3 | |         // nothing to do for this values
4 | |         } break;
  | |_________________^
  |
  = help: Add `break` or comment `/* falls through */` to your case statement
error[no-fallthrough]: Fallthrough is not allowed
 --> test.js:2:5
  |
2 |       case 0: {
  |  _____^
3 | |         // nothing to do for this values
4 | |         } break;
  | |_________________^
  |
  = help: Add `break` or comment `/* falls through */` to your case statement
Found 2 problems
```

### Additional Info

This is using dlint.

### Version

deno 1.11.1 (release, x86_64-pc-windows-msvc)
v8 9.1.269.35
typescript 4.3.2

dlint 0.6.0

**Comment by magurotuna** [comment]
Thanks for the bug report! I'm fixing the false positive.
On the other hand, in my environment, your second bug (the same error reported twice) isn't reproduced. I tried both deno v1.11.2 and dlint v0.6.0.
Does this bug happen for this specific case? Or this happens for all diagnostics?

**Comment by lawrence-dol** [comment]
@magurotuna : Thanks for fixing this issue so quickly!

The duplicate report bug happens consistently for at least most diagnostics. I'll open a new issue for this, since it can create a lot of noise when linting code that was never linted before.
